employees/common/strings.py

```diff
@@ -94,11 +94,12 @@ class ProjectReportDetailStrings(NotCallableMixin, Enum):
 
 
 class ReportValidationStrings(NotCallableMixin, Enum):
+    WORK_HOURS_MINUTES_ARE_INDIVISIBLE_BY_FIFTEEN = _("Possible values for minutes are 0, 15, 30 and 45")
```

Comment:
I would give it more generic name, because the requirement might change in the future, then the message will have to change, and then the enum item name will have to change too.

Reply:
Okay, done :)